content/episodes/2019-02-04-S04E05-the-surveillance-economy.md

```diff
@@ -0,0 +1,199 @@
+---
+date: 2019-02-04T04:55:00-08:00
+title: "The Surveillance Economy"
+ogtitle: "The Surveillance Economy"
+description: |
+    In her new book, The Age of Surveillance Capitalism, Harvard Business School’s [Shoshana Zuboff](https://twitter.com/shoshanazuboff/) argues that tech companies — like Google and Facebook — collect so much personal data for profit, that they’re changing the fundamentals of our economy and way of life. And now these companies are learning to shape our behavior to better serve their business goals. Shoshana joins [Manoush Zomorodi](https://twitter.com/manoushz/) to explain what this all means for us.
+
+    We then explore whether or not it’s time to end our relationship with corporate spies. OG advice columnist [Dear Abby](https://twitter.com/dearabby/) gives us some tips to start with. We chat with philosopher [S. Matthew Liao](https://twitter.com/smatthewliao/). He asks if we have a moral duty to quit Facebook. [Alice Marwick](https://twitter.com/alicetiara/) explains why most people won’t leave the social network. And journalist [Nithin Coca](https://twitter.com/excinit/) tells us what it was like for him to quit both Facebook and Google. Spoiler alert: it wasn’t easy, but he has no regrets.
+ogdescription: "Explore the surveillance economy, the companies that participate in it, and its impact on you."
+number: 28
+season: 4
+seasonepisode: 5
+url: /season4/episode5/
+embed: "c70f8ee2"
+mp3: "https://dts.podtrac.com/redirect.mp3/audio.simplecast.com/c70f8ee2.mp3"
+categories: "episodes"
+host: "Manoushz"
+shownotes: |
+    [Shoshana Zuboff](https://www.shoshanazuboff.com/new/about/) is the author of [The Age of Surveillance Capitalism](https://www.amazon.com/Age-Surveillance-Capitalism-Future-Frontier/dp/1610395697/ref=sr_1_1?ie=UTF8&qid=1548643852&sr=8-1&keywords=The+Age+of+Surveillance+Capitalism/).
```

Comment:
We should shorten the Amazon URL to `https://www.amazon.com/Age-Surveillance-Capitalism-Future-Frontier/dp/1610395697/` without the extra parameters.